podman/domain/containers.py

```diff
@@ -25,7 +25,7 @@ class Container(PodmanResource):
     def name(self):
         """str: Returns container's name."""
         with suppress(KeyError):
-            return self.attrs["Name"].lstrip("/")
+            return self.attrs["Names"][0].lstrip("/")
```

Comment:
The ci is failing trying to check the name property of a container. I wonder if we need to handle both cases

e.g.

 if 'Name' in attrs:
    return self.attrs["Name"].lstrip("/")
else:
    return self.aters["Names"][0].lstrip("/")